core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestTransformCorrelatedGlobalAggregationWithProjection.java

```diff
@@ -287,4 +289,37 @@ public void testWithPreexistingMask()
                                                         .right(project(ImmutableMap.of("non_null", expression(TRUE)),
                                                                 values(ImmutableMap.of("a", 0, "mask", 1)))))))));
     }
+
+    @Test
+    public void rewritesOnSubqueryWithBoolOr()
+    {
+        tester().assertThat(new TransformCorrelatedGlobalAggregationWithProjection(tester().getPlannerContext()))
+                .on(p -> p.correlatedJoin(
+                        ImmutableList.of(p.symbol("corr")),
+                        p.values(p.symbol("corr")),
```

Comment:
This version of the `symbol(...)` method is deprecated. Please use the one that takes a type explicitly.